src/LottieAnimation.ts

```diff
@@ -26,6 +27,7 @@ export class LottieAnimation extends Script {
 	private _clip: TypeAnimationClip;
 	private _clipEndCallbacks: Object = {};
 	private _autoPlay: boolean = false;
+	private _pivotVector: Vector2 = new Vector2();
```

Comment:
Static can be used here.

Reply:
![image](https://user-images.githubusercontent.com/7768919/146331987-88d823f8-7c91-420c-be45-4691578cb6d9.png)
This is how the sprite sets the pivot.

Reply:
Fine.